app/src/components/GrantReviewList.vue

```diff
@@ -5,9 +5,9 @@
       <li v-for="grant in sortedGrants" :key="grant.id.toString()">
         <GrantCard
           :id="grant.id"
-          :name="(grantMetadata && grantMetadata[grant.metaPtr]?.name) || '...'"
+          :name="(grantMetadata && grantMetadata[metadataId(grant.metaPtr)]?.name) || '...'"
           :ownerAddress="grant.owner"
-          :imgurl="(grantMetadata && grantMetadata[grant.metaPtr]?.logoURI) || '/placeholder_grant.svg'"
+          :imgurl="(grantMetadata && grantMetadata[metadataId(grant.metaPtr)]?.logoURI) || '/placeholder_grant.svg'"
```

Comment:
would it make sense to wrap grantMetadata[metadataId(grant.metaPtr)] into a variable called `metadata` and return from setup function just so that the templating is a lil easier to read ?

(more for readability)

Reply:
Hm, maybe I'm misunderstanding but IMO it becomes less readable, since instead of a 1 line change, now you need to have a metadata array generated by looping through each grant passed in `sortedGrants`, and keep track of an index in the `v-for` elements to make sure you're accessing the corresponding element in the metadata array. And since grant order changes by sorting order, you need to account for that also and regenerate the `metadata` array each time